### ISSUE TITLE
Support for window functions

### ISSUE THREAD

**Issue by Photonios** [SectorLabs/django-postgres-extra#14]
https://www.postgresql.org/docs/9.1/static/tutorial-window.html

**Comment by adamchainz** [comment]
These are in Django 2.0: https://docs.djangoproject.com/en/2.0/releases/2.0/

**Comment by Photonios** [comment]
@adamchainz Oh that’s great! That means this issue will soon be obsolete once it has been tested with Django 2.0. Or do you think backporting it is worth it?

**Comment by adamchainz** [comment]
I don't think it would be possible to backport in a third party library, it required changes to the ORM internals. This issue would then be obsolete :)
